app/Http/Controllers/Api/TagsController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Rogue\Http\Controllers\Api;
+
+use Rogue\Http\Requests\TagsRequest;
+use Rogue\Repositories\PostRepository;
+use Rogue\Http\Transformers\PostTransformer;
+
+class TagsController extends ApiController
+{
+    /**
+     * The post service instance.
+     *
+     * @var Rogue\Repositories\PostRepository
+     */
+    protected $post;
+
+    /**
+     * @var \League\Fractal\TransformerAbstract;
+     */
+    protected $transformer;
+
+    /**
+     * Create a controller instance.
+     *
+     * @param  PostContract $posts
+     * @return void
+     */
+    public function __construct(PostRepository $post)
+    {
+        $this->post = $post;
+        $this->transformer = new PostTransformer;
+    }
+
+    public function store(TagsRequest $request)
```

Comment:
One thing I did for the `PostController` store method was I created a trait `PostRequests` that had the `store()` method in it and then used that train in both `PostController.php` and `Api/PostController.php` that way the logic could be shared between the two controllers and only live in one place.